packages/gestalt/src/InternalSheet.js

```diff
@@ -28,8 +29,34 @@ type InternalSheetProps = {|
   children: NodeOrRenderProp,
   closeOnOutsideClick: boolean,
   footer: NodeOrRenderProp,
-  heading: NodeOrRenderProp,
+  heading?: string,
   onAnimationEnd: ?({| animationState: 'in' | 'out' |}) => void,
+  dismissConfirmation?: {|
```

Comment:
There's no need to keep this level of nesting — spread these props to SheetConfirmationPopover in InternalSheet, keep the props type here more flat

Reply:
DONE